crates/driver/src/infra/api/routes/solve/dto/auction.rs

```diff
@@ -113,7 +115,8 @@ impl Auction {
                     })
                 })
                 .try_collect()?,
-            gas_price: self.effective_gas_price.into(),
+            // TODO: Populate hardcoded value.
+            gas_price: eth::U256::from(0).into(),
```

Comment:
Given the new auction model, I agree that this shouldn't be sent by the `autopilot`.